### PR TITLE
Rmm v2v h2v discoverytool

### PR DESCRIPTION
**Pull request by godurga**
1. This branch contains VMware and HyperV discovery tools. 
2. We are temporarily keeping older files (config.ini,l checksum.txt, discoveryTool of VMware Discovery Tool). Parallel to VMware and HyperV folder. 
3. We will delete those temporarily files as soon as we go GA. 
4.   3 Readme files are added: Readme for VMware, Hyper-V, and VPC Tools.

### CODE REVIEW

**Comment by rahulippar** [comment]
Approved.

**Review by rahulippar — COMMENTED** [review]
Approved

**Review by go-davidng — APPROVED** [review]
approved

File: v2v-discovery-tool-rmm/HyperV/README.md [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+		
+To check the Discovery Tool version, use the following command:
+
+```Shell
+$ ./discoveryTool --version
+```
+
+#### SCVMM password
+
+- Discovery Tool prompts you for the administrator password of the SCVMM server.
```
[the remark this answers is not in the document]
Comment (rahulippar) [reply v2v-discovery-tool-rmm/HyperV/README.md:117]: Updated second bullet point under SCVMM password section to incorporate mentioned suggestion  as follows:
`Discovery Tool communicate to SCVMM server and to the VM over SSH (port 22) and thus only IP address(es) that are open on SSH port 22 will be saved to the csv file.`

File: v2v-discovery-tool-rmm/HyperV/README.md [hunk 2]
```diff
@@ -0,0 +1,124 @@
+# Hyper-V Single Cluster Discovery Tool with SCVMM
+
+## Discovering the Hyper-V guest virtual machine(s) using the Discovery Tool
+
+#### Overview
+
+The Discovery tool automates the discovery of Hyper-V virtual machine(s) settings for the RackWware RMM migration tool. By using the System Center Virtual Machine Manager (SCVMM) PowerShell cmdlets, the following attributes are captured and saved to a **csv** file:
```
[the remark this answers is not in the document]
Comment (rahulippar) [reply v2v-discovery-tool-rmm/HyperV/README.md:7]: Created separate section **SCVMM PowerShell Cmdlets** to incorporate mentioned suggestion in readme. Check out following screenshot.

image

**Pull request closed without merging**